noether_simulator/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(noether_simulator)
+SET(CMAKE_CXX_FLAGS "-std=c++0x")
+
+find_package(VTK 7.1 CONFIG)
+find_package(VTK 7.1 REQUIRED)
+include(${VTK_USE_FILE})
+
+find_package(Eigen3 REQUIRED)
+
+find_package(PCL 1.8 REQUIRED)
+
+find_package(Boost REQUIRED COMPONENTS system)
+
+find_package(catkin REQUIRED cmake_modules
+    message_generation
+    mesh_segmenter
+    tool_path_planner
+    path_sequence_planner
+    vtk_viewer
+    sensor_msgs
+    geometry_msgs
+    roscpp
+    pcl_msgs
+    std_srvs
+    pcl_conversions
+    dynamic_reconfigure
+    genmsg
+    actionlib
+    actionlib_msgs
+)
```

Comment:
I know some of this can be removed, I suspect a lot of it. Just taking a guess, after removing the ROS node these can probably be removed.
```
    message_generation
    mesh_segmenter
    path_sequence_planner
    sensor_msgs
    geometry_msgs
    roscpp
    pcl_msgs
    std_srvs
    pcl_conversions
    dynamic_reconfigure
    genmsg
    actionlib
    actionlib_msgs
```

I see that you need some of them for the tests. In that case you can add them below.